### PR TITLE
Uploaded In Assignment 7 & 8

### PR DESCRIPTION
**Pull request by Ashwin2812**

### CODE REVIEW

File: Ashwin/HTML/07_Assignment/Homepage.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+.nav-link {
+    color:white;
+}
+.nav-link:hover {
+    color: red;
+    font-weight: bold;
+}
+.nav-item:hover {
+    background-color: white;
+}
```
Comment (shubhamdalal49) [inline Ashwin/HTML/07_Assignment/Homepage.css]: add extra line at the end

File: Ashwin/HTML/07_Assignment/Personalstyle.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    padding: 20px;
+    background-color: blanchedalmond;
+}
+.form2 {
+    height: 600px;
+    width: 100%;
+    border: 1px solid red;
+    padding: 20px;
+    background-color: antiquewhite;
+}
```
Comment (shubhamdalal49) [inline Ashwin/HTML/07_Assignment/Personalstyle.css]: add extra line at the end

File: Ashwin/HTML/08_Assignment/08_Ass.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    display: flex;
+    justify-content:stretch;
+}
+.ImageMain > img {
+    width: 200px;
+    height: 200px;
+}
+.ImageMain > img, .ImageMain > p {
+    padding: 0 5px;
+}
```
Comment (shubhamdalal49) [inline Ashwin/HTML/08_Assignment/08_Ass.css]: add extra line at the end

File: Ashwin/HTML/08_Assignment/08_Assign.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                    voluptatibus, quia, quos excepturi modi atque, repellat aliquid qui. Esse incidunt tenetur veritatis! Lorem, ipsum dolor sit amet
+                    consectetur adipisicing elit. Nulla distinctio saepe consequuntur ex repudiandae nihil iste dolorum voluptatibus, quia, quos excepturi modi
+                    atque, repellat aliquid qui. Esse incidunt tenetur veritatis! Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla distinctio
+                    saepe consequuntur ex repudiandae nihil iste dolorum voluptatibus, quia, quos excepturi modi atque, repellat aliquid qui. Esse incidunt
+                    tenetur veritatis!
+                </p>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (shubhamdalal49) [inline Ashwin/HTML/08_Assignment/08_Assign.html]: add extra line at the end

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

File: Ashwin/HTML/07_Assignment/loginstyle.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
+.bigbox {
+    height: 400px;
+    width: 750px;
+    background-image: url("Images/2.jpeg");
+    background-size: cover;
+    margin-top: 250px;
+    padding: 30px;
+    border-radius: 20px;
+}
+.box {
```
Comment (Priyankashelke2411) [inline Ashwin/HTML/07_Assignment/loginstyle.css:11]: line required after every class end statement

File: Ashwin/HTML/07_Assignment/loginstyle.css [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    height: 350px;
+    width: 700px;
+    border: 1px solid black;
+    padding: 30px;
+    border-radius: 15px;
+}
+h2 {
```
Comment (Priyankashelke2411) [inline Ashwin/HTML/07_Assignment/loginstyle.css:19]: check everywhere line required after every class

File: Ashwin/HTML/08_Assignment/08_Ass.css [hunk 7]
```diff
@@ -0,0 +1,17 @@
+.container {
+    width: 100%;
+    height: auto;
+    background-color: cyan;
+    border: solid 2px red;
+}
+.ImageMain {
+    display: flex;
+    justify-content:stretch;
```
Comment (Priyankashelke2411) [inline Ashwin/HTML/08_Assignment/08_Ass.css]: space required after :

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**